backtrace-library/src/main/java/backtraceio/library/breadcrumbs/BacktraceBreadcrumbs.java

```diff
@@ -103,8 +106,14 @@ private void registerAutomaticBreadcrumbReceivers() {
         }
 
         backtraceBroadcastReceiver = new BacktraceBroadcastReceiver(this);
-        context.registerReceiver(backtraceBroadcastReceiver,
-                backtraceBroadcastReceiver.getIntentFilter());
+
+        if(Build.VERSION.SDK_INT >= 33) {
```

Comment:
maybe let's extract whole if-else to separate method `registerBroadcastreceiver`

Reply:
https://github.com/backtrace-labs/backtrace-android/pull/105/files - done here. I moved to another diff to avoid mering it to the release branch.